pos_access_right/models/pos_config.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, api
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    group_pos_negative_qty = fields.Many2one(
```

Comment:
if you want to keep field names shorter, you can drop "_pos_"... we know we are talking about "pos" things here ;)